library/Zend/Code/Reflection/ClassReflection.php

```diff
@@ -177,12 +177,17 @@ public function getMethods($filter = -1)
         return $methods;
     }
 
+    /**
+     * Returns an array of reflection classes of traits used by this class.
+     *
+     * @return array|null
+     */
     public function getTraits()
     {
         $vals = array();
         $traits = parent::getTraits();
-        if (! $traits) {
-            return;
+        if ($traits === null) {
+            return null;
```

Comment:
`return null;` was removed from classes because it is the same as only `return;`. See #7014.